mgmtsystem_probability/model/mgmtsystem_probability.py

```diff
@@ -0,0 +1,47 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 - Present
+#    Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.tools.translate import _
+from osv import fields, orm
+
+_CATEGORIES = [
+    ("hazard", _("hazard")),
+    ("security", _("security")),
+]
```

Comment:
`_("...")` wont work the way you expect it to here.

The function requires a `context` variable to be defined in the local frame.

Here is does not, since it is run when Odoo imports the module folder (therefore `__init__.py` and `mgmtsystem_probability.py`).

A way around this is to have a function reference in the `fields.selection(...)` declaration.

```python
_columns = {
    "category": fields.selection(
        lambda self, *a, **kw: self._category_selection(*a, **kw),
        "Category", required=True,
    ),
}

def _category_selection(self, cr, uid, context=None):
    return [
        ("hazard", _("hazard")),
        ("security", _("security")),
    ]
```

The use of lambda is to the function can be inherited and categories added.